app/src/org/commcare/android/database/global/GlobalDatabaseUpgrader.java

```diff
@@ -50,6 +54,15 @@ private boolean upgradeOneTwo(SQLiteDatabase db, int oldVersion, int newVersion)
     }
 
     private boolean upgradeTwoThree(SQLiteDatabase db) {
```

Comment:
@ctsims It's worth noting that since upgradeTwoThree() was already merged into master (although NOT released yet), anyone who has the build from master installed on their device (i.e. us, and anyone trying out multiple apps or using it for some other special reason) will have to completely uninstall CommCare for this to work. If we don't like that, I could change the dbVersion again to 4, and put this stuff in upgradeThreeFour(). Let me know what you think.